alicloud/resource_alicloud_fc_trigger_test.go

```diff
@@ -116,7 +118,7 @@ variable "account" {
   default = "1204663572767468"
 }
 variable "name" {
-  default = "test-alicloud-fc-trigger"
```

Comment:
This is test case also hardcode account id. Can you remove it and get account id automatically?

Reply:
I will merge the code with the master to fix that.